pkg/services/ngalert/api/tooling/definitions/provisioning.go

```diff
@@ -0,0 +1,9 @@
+package definitions
+
+// AlertingFileExport is the full provisioned file export.
+// swagger:model
+type AlertingFileExport struct {
+	APIVersion int64                      `json:"apiVersion" yaml:"apiVersion"`
+	Groups     []AlertRuleGroupExport     `json:"groups,omitempty" yaml:"groups,omitempty"`
+	Policies   []NotificationPolicyExport `json:"policies,omitempty" yaml:"policies,omitempty"`
+}
```

Comment:
Alertmanager (and therefore provisioning) will reject policies which reference contact points that aren't defined. So I think since this file doesn't contain any contact points, the files it produces are not importable. Should we implement some form of export for contact points first? Same goes for mute timings.

Reply:
Correct, any referenced contact points and mute timings referenced need to already exist otherwise if you try to import the exported policy it will fail. This makes it less useful than it could be with the other export endpoints but still functional.

The rest of the exports [are on the way](https://github.com/grafana/grafana/issues/70004) and I'll be working on Contact Points and Mute Timings next.